Recrutify/Recrutify.Host/CustomAttribute/ParseJsonToDict.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Recrutify.Host.CustomAttribute
+{
+    public class ParseJsonToDict
+    {
+        [JsonProperty(PropertyName = "projectId")]
```

Comment:
you can Pass JsonSerializerOptions() with PropertyNameCaseInsensitive = true to DeserializeObject method and remove this attributes